heltour/tournament/tasks.py

```diff
@@ -22,89 +49,95 @@
 import reversion
 import textwrap
 import time
-import sys, traceback
+import sys
+import traceback
 
 logger = get_task_logger(__name__)
 
 UsernamesQuerySet = ValuesQuerySet[Player, Dict[str, str]]
 
+
 def to_usernames(users: UsernamesQuerySet) -> List[str]:
-    return [p['lichess_username'] for p in users if p['lichess_username'].strip()]
+    return [p["lichess_username"] for p in users if p["lichess_username"].strip()]
 
 
 def just_username(qs: QuerySet[Player]) -> UsernamesQuerySet:
-    return qs \
-        .order_by('lichess_username') \
-        .values('lichess_username') \
-        .distinct()
+    return qs.order_by("lichess_username").values("lichess_username").distinct()
```

Comment:
This feels like something that might be nicely added to the QuerySetManager object for Player objects, rather than a method, but we don't have to do that right now.